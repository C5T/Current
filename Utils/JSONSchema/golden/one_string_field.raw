{"ObjectOrArray":{"fields":{"a":{"String":{"values":{"b":1},"instances":1,"nulls":0}}},"instances":1,"nulls":0}}