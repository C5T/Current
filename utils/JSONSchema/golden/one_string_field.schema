// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(a, std::string);
};

using Schema = Schema_Object;
