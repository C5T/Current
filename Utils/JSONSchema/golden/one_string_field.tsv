Field	Type	Set	Unset/Null	Values	Details
Schema	Object	1	0	1 field	a
Schema.a	String	1	0	1 distinct value	`b` : 1
